htdocs/web_portal/views/admin/add_service_type.php

```diff
@@ -6,8 +6,14 @@
         <input type="text" value="" name="Name" class="input_input_text">
         <span class="input_name">Description</span>
         <input type="text" value="" name="Description" class="input_input_text">
-
+        <span class="input_name" style="">
```

Comment:
Can we remove style attribute if it is unused ?

Reply:
`style=""` is used to a limited extent in other parts of the codebase - seemingly limited to pages around ServiceGroups.

We should determine the effect of removing `style=""` from the codebase and do it in one go if that's determined to be the best thing. Otherwise, we should add comments why the empty style block is needed.

See #445